### PR TITLE
Bump GraphiQL version

### PR DESCRIPTION
**Pull request by steveluscher**

### CODE REVIEW

**Comment by steveluscher** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #1** <a href="https://app.graphite.dev/github/pr/buffalojoec/solana-graphql/1?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #2** <a href="https://app.graphite.dev/github/pr/buffalojoec/solana-graphql/2?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
      * **PR #3** <a href="https://app.graphite.dev/github/pr/buffalojoec/solana-graphql/3?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #4** <a href="https://app.graphite.dev/github/pr/buffalojoec/solana-graphql/4?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #5** <a href="https://app.graphite.dev/github/pr/buffalojoec/solana-graphql/5?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #6** <a href="https://app.graphite.dev/github/pr/buffalojoec/solana-graphql/6?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
              * **PR #7** <a href="https://app.graphite.dev/github/pr/buffalojoec/solana-graphql/7?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                * **PR #8** <a href="https://app.graphite.dev/github/pr/buffalojoec/solana-graphql/8?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                  * **PR #10** <a href="https://app.graphite.dev/github/pr/buffalojoec/solana-graphql/10?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                    * **PR #11** <a href="https://app.graphite.dev/github/pr/buffalojoec/solana-graphql/11?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                      * **PR #12** <a href="https://app.graphite.dev/github/pr/buffalojoec/solana-graphql/12?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                  * **PR #9** <a href="https://app.graphite.dev/github/pr/buffalojoec/solana-graphql/9?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/buffalojoec/solana-graphql/2?utm_source=stack-comment).

**Review by buffalojoec — APPROVED** [review]
I remember adding this for some odd reason..?

**Comment by steveluscher** [comment]
> I remember adding this for some odd reason..?
> 
> 

Yeah, you're right. It's a _dev_ dependency of `@graphql/toolkit` but they didn't mark it as one or something.

File: package.json [hunk 1]
```diff
@@ -27,6 +26,7 @@
         "autoprefixer": "^10.0.1",
         "eslint": "^8",
         "eslint-config-next": "14.0.1",
+        "graphql-ws": "^5.14.3",
```
Comment (steveluscher) [inline package.json:29]: I put it back, but in `devDependencies`.

**Comment by steveluscher** [comment]
### Merge activity

* **Dec 20, 11:33 PM**: @@steveluscher started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/buffalojoec/solana-graphql/2).
* **Dec 20, 11:34 PM**: @@steveluscher merged this pull request with [Graphite](https://app.graphite.dev/github/pr/buffalojoec/solana-graphql/2).

**Pull request merged**